src/componenets/landing/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Grid } from "@mui/material";
+import CustomCard from "../../common/card";
+import Header from "../header";
+
+const Landing = () => {
+  return (
+    <div>
+      <Header />
+      <Grid
+        container
+        sx={{
+          display: "flex",
+          justifyContent: "space-between",
+          marginTop: "10%",
+        }}
+      >
+        <Grid item md={3} mt="2%">
+          <CustomCard text=" Morning Tea " />
+        </Grid>
+        <Grid item md={3} mt="2%">
+          <CustomCard text="Afternoon !!" />
+        </Grid>
+        <Grid item md={3} mt="2%">
+          <CustomCard text="Eveneing Tea" />
+        </Grid>
```

Comment:
we can avoid duplication of code here like 
const events = ["tea", "lunch", "tea2"]
events.map(item=><Grid item md={3} mt="2%">
          <CustomCard text="item!" />
        </Grid>)